bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -521,11 +572,17 @@ public boolean execute(String script) {
 	// Feature in WebView2. ExecuteScript works regardless of IsScriptEnabled setting.
 	// Disallow programmatic execution manually.
 	if (!jsEnabled) return false;
-
+	waitForWebviewInitialization();
 	IUnknown completion = newCallback((long result, long json) -> COM.S_OK);
-	int hr = webView.ExecuteScript(stringToWstr(script), completion);
+	lastWebViewTask.join().ExecuteScript(stringToWstr(script), completion);
 	completion.Release();
-	return hr == COM.S_OK;
+	return true;
```

Comment:
So, like Heiko and I agreed, we always send a true since we handle the request using a future, so i might not be complete by the time we would have to return something. It is the adapation of the webkit implementation which also does it similarly.

Reply:
Reverted